resources/rpm/project.spec.erb

```diff
@@ -248,6 +251,18 @@ fi
   <%- end -%>
 <%- end -%>
 
+<%- if @platform.is_aix? || (@platform.is_el? && @platform.os_version.to_i == 4) -%>
+%triggerpostun -- <%= @name %>
+# old versions of vanagon generated an unconditional service stop in the preun. This
+# means that postinstall actions that mess with the service need to be run AFTER that
+# step. To do so, we can run them in the "triggerpostun" when an upgrade is happening.
+
+# Run postinstall scripts on upgrade if defined
+if [ "$2" -gt 0 ] ; then
+  <%= get_postinstall_actions("upgrade") %>
+fi
+<%- end -%>
```

Comment:
(nitpick) indentation in this block is off

Reply:
Updated